<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsTableEditorBase</class>
 <widget class="QMainWindow" name="QgsTableEditorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2180</width>
    <height>609</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="mViewFrame">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="mStatusBar"/>
  <widget class="QMenuBar" name="mMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2180</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Table">
    <property name="title">
     <string>&amp;Table</string>
    </property>
    <widget class="QMenu" name="menuInsert_Rows">
     <property name="title">
      <string>Insert Rows</string>
     </property>
     <addaction name="mActionInsertRowsAbove"/>
     <addaction name="mActionInsertRowsBelow"/>
    </widget>
    <widget class="QMenu" name="menuInsert_Columns">
     <property name="title">
      <string>Insert Columns</string>
     </property>
     <addaction name="mActionInsertColumnsBefore"/>
     <addaction name="mActionInsertColumnsAfter"/>
    </widget>
    <addaction name="menuInsert_Rows"/>
    <addaction name="menuInsert_Columns"/>
    <addaction name="mActionDeleteRows"/>
    <addaction name="mActionDeleteColumns"/>
    <addaction name="separator"/>
    <addaction name="mActionMergeSelected"/>
    <addaction name="mActionSplitSelected"/>
    <addaction name="separator"/>
    <addaction name="mActionIncludeHeader"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="mActionSelectAll"/>
    <addaction name="mActionSelectRow"/>
    <addaction name="mActionSelectColumn"/>
    <addaction name="separator"/>
    <addaction name="mActionClear"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mActionImportFromClipboard"/>
    <addaction name="separator"/>
    <addaction name="mActionClose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Table"/>
  </widget>
  <action name="mActionSetForeground">
   <property name="text">
    <string>Set Foreground Color…</string>
   </property>
  </action>
  <action name="mActionSetBackground">
   <property name="text">
    <string>Set Background Color…</string>
   </property>
  </action>
  <action name="mActionInsertRowsAbove">
   <property name="text">
    <string>Rows Above</string>
   </property>
  </action>
  <action name="mActionInsertRowsBelow">
   <property name="text">
    <string>Rows Below</string>
   </property>
  </action>
  <action name="mActionInsertColumnsBefore">
   <property name="text">
    <string>Columns Before</string>
   </property>
  </action>
  <action name="mActionInsertColumnsAfter">
   <property name="text">
    <string>Columns After</string>
   </property>
  </action>
  <action name="mActionDeleteRows">
   <property name="text">
    <string>Delete Rows</string>
   </property>
  </action>
  <action name="mActionDeleteColumns">
   <property name="text">
    <string>Delete Columns</string>
   </property>
  </action>
  <action name="mActionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="mActionSelectRow">
   <property name="text">
    <string>Select Row</string>
   </property>
  </action>
  <action name="mActionSelectColumn">
   <property name="text">
    <string>Select Column</string>
   </property>
  </action>
  <action name="mActionImportFromClipboard">
   <property name="text">
    <string>Import Content from Clipboard</string>
   </property>
  </action>
  <action name="mActionClose">
   <property name="text">
    <string>Close Editor</string>
   </property>
  </action>
  <action name="mActionSetRowHeight">
   <property name="text">
    <string>Set Row Height…</string>
   </property>
  </action>
  <action name="mActionSetColumnWidth">
   <property name="text">
    <string>Set Column Width…</string>
   </property>
  </action>
  <action name="mActionClear">
   <property name="text">
    <string>Clear Cell(s)</string>
   </property>
  </action>
  <action name="mActionIncludeHeader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Include Header Row</string>
   </property>
  </action>
  <action name="mActionMergeSelected">
   <property name="text">
    <string>Merge Selected Cells</string>
   </property>
  </action>
  <action name="mActionSplitSelected">
   <property name="text">
    <string>Split Selected Cells</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
